.changeset/old-islands-invite.md

```diff
@@ -0,0 +1,5 @@
+---
+"open-next": patch
+---
+
+Fix Image Optimization Support for Next@14.1.1
```

Comment:
We probably don't want to apply the changeset now. We usually let the PR merge into main, run the e2e, and once that passes, we do a separate changeset PR.

Reply:
It should be fine actually, changesets get merged before release and the changebot don't release until we merge the version packages PR.

Reply:
oh right, I forgot.